Build/OpenDDSharp.Build/Tasks/SetupThirdPartyTask.cs

```diff
@@ -93,7 +93,7 @@ public override void Run(BuildContext context)
             if (BuildContext.IsLinux || BuildContext.IsOSX)
             {
                 var configurePath = System.IO.Path.Combine(_clonePath.FullPath, "configure");
-                var arguments = " -v --ace-github-latest --no-test --no-debug --optimize --install-origin-relative --prefix=/usr/lib";
+                var arguments = " -v --ace-github-latest --doc-group3 --no-test --no-debug --optimize --install-origin-relative --prefix=/usr/lib";
```

Comment:
It is or ace-github-latest which uses the latest commit of the ace6tao2 branch, or doc-group3 which is a release, there is no support for master of the ACE/TAO repo as part of the configure script